roles/openshift_storage_ceph/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+openshift_storage_ceph_timeout: 300
+openshift_storage_ceph_mon_count: 1
+openshift_storage_ceph_use_host_network: False
+openshift_storage_ceph_dirs: "/var/lib/rook"
+openshift_storage_ceph_dashboard: True
+openshift_storage_ceph_dashboard_url_prefix: "/ceph-dashboard"
+openshift_storage_ceph_image: "{{ 'rhceph/rhceph-3-rhel7' | quote if deployment_type == 'openshift-enterprise' else 'ceph/ceph' | quote }}"
```

Comment:
This should use `lib_utils_oo_oreg_image`, see https://github.com/openshift/openshift-ansible/blob/master/roles/etcd/defaults/main.yaml#L11-L18